hopsworks-common/src/main/java/io/hops/hopsworks/common/util/SparkConfigurationUtil.java

```diff
@@ -147,6 +155,7 @@
                             experimentType != null)));
 
     if(experimentType != null) {
+      addToSparkEnvironment(sparkProps,"LC_ALL", "C", settings);
```

Comment:
@robzor92 are these properties somewhere documented?

Reply:
I ended up skipping setting LC_ALL. This setting is supposed to force all locale settings to be the same, for example en_US.UTF-8. I had to set LC_ALL=C to get rocm to work, now it seems I can skip setting it. However we have issues with locale on other installations (TF doesn't like swedish) so it needs to be managed at some point. For example we should maybe set all locale aspects to be en_US.UTF-8 in chef. Added link to docs for CUDA_VISIBLE_DEVICES and HIP_VISIBLE_DEVICES.